src/Okta.Sdk/Client/ApiClient.cs

```diff
@@ -476,7 +476,7 @@ internal RestClient GetConfiguredClient(IReadableConfiguration configuration)
             {
                 clientOptions.Proxy = _proxy;
             }
-            else if (configuration.Proxy != null)
+            else if (configuration.Proxy != null && configuration.UseProxy == true)
```

Comment:
It seems UseProxy is nullable, should we add a null check here?

Reply:
If UseProxy is null in the comparison then it will be `null == true` which evaluates to false.

Reply:
Right!